core/src/main/resources/hudson/security/FullControlOnceLoggedInAuthorizationStrategy/config.jelly

```diff
@@ -0,0 +1,7 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson"
+  xmlns:f="/lib/form">
+  <f:entry field="allowAnonymousRead">
```

Comment:
Polite to add a `help-allowAnonymousRead.html` giving a fuller explanation than the `title`.